Dashboard/Views/ConnectionWindow.axaml

```diff
@@ -0,0 +1,93 @@
+            <MenuItem Header="Console"/>
+            <MenuItem Header="About"/>
+            <MenuItem Header="Help"/>
+        </Menu>
+
+        <!-- Bottom Status Panel -->
+        <Grid DockPanel.Dock="Bottom" Background="LightGray"  Height="60">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="1*"/>
+                <!-- Left -->
+                <ColumnDefinition Width="1*"/>
+                <!-- Center -->
+                <ColumnDefinition Width="1*"/>
+                <!-- Right -->
+            </Grid.ColumnDefinitions>
+
+            <!-- Left Side: Car Number -->
+            <StackPanel Grid.Column="0" Orientation="Horizontal" VerticalAlignment="Center">
+                <TextBlock Text="Car Number" VerticalAlignment="Center" Margin="20,0"/>
+                <TextBox Width="150" Margin="5,0" VerticalAlignment="Center"/>
+            </StackPanel>
+
+            <!-- Center: UTC Time and SN -->
+            <StackPanel Grid.Column="1" Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+                <TextBlock Text="UTC Time" VerticalAlignment="Center"/>
+                <TextBlock Text="SN: 987654321" Margin="30,0,0,0" VerticalAlignment="Center"/>
+            </StackPanel>
+
+            <!-- Right Side: Connection and Heart Beat -->
+            <StackPanel Grid.Column="2" Orientation="Horizontal" HorizontalAlignment="Right" VerticalAlignment="Center">
+                <RadioButton Content="Connection" GroupName="Status" VerticalAlignment="Center"/>
+                <RadioButton Content="Heart Beat" GroupName="Status" Margin="10,0" VerticalAlignment="Center"/>
+            </StackPanel>
+        </Grid>
+
+        <!-- Main Content Grid -->
+        <Grid DockPanel.Dock="Top">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="2*"/>
+                <ColumnDefinition Width="3*"/>
+            </Grid.ColumnDefinitions>
+
+            <StackPanel Grid.Column="0" Margin="20">
+                <!-- Connection Type -->
+                <TextBlock Text="Connection Type" Margin="0, 10, 0, 5"/>
+                <ComboBox Width="200">
+                    <ComboBoxItem>Serial Port</ComboBoxItem>
+                    <ComboBoxItem>Ethernet</ComboBoxItem>
+                    <ComboBoxItem>USB</ComboBoxItem>
+                    <ComboBoxItem>Wi-Fi</ComboBoxItem>
+                </ComboBox>
+
+                <!-- Serial Port -->
+                <TextBlock Text="Serial Port" Margin="0, 10, 0, 5"/>
+                <ComboBox Width="200">
+                    <ComboBoxItem>COM1</ComboBoxItem>
+                    <ComboBoxItem>COM2</ComboBoxItem>
+                    <ComboBoxItem>COM3</ComboBoxItem>
+                    <ComboBoxItem>COM4</ComboBoxItem>
+                    <ComboBoxItem>COM5</ComboBoxItem>
+                </ComboBox>
+
+                <!-- IP Address -->
+                <TextBlock Text="IP Address" Margin="0, 15, 0, 5"/>
```

Comment:
Each of these sections should be hidden until the connection type is selected above, e.g. if serial port is selected hide everything besides the serial port drop down.

Reply:
you didnt fix this